### PR TITLE
feat(cache): Language headers

### PR DESCRIPTION
**Pull request by MetecanAydin**
Update:
- Add logic to manually set content-language header in cache server response
- Add unit tests

### CODE REVIEW

File: apps/cache-manager/src/app.ts [hunk 1]
```diff
@@ -41,6 +49,7 @@ app.post('/:port/prerender', async (req, res) => {
       component = response.data
     }
 
+    res.set(headers)
```
Comment (JLingardAmido) [inline apps/cache-manager/src/app.ts:52]: Do we need to send back content language?

File: apps/cache-manager/src/app.ts [hunk 2]
```diff
@@ -17,10 +17,18 @@ app.post('/:port/prerender', async (req, res) => {
 
     const remoteName = req.get('remote-name') || ''
 
+    const languageHeader = req.headers['content-language'] || 'en-GB'
+    const headers = {
+      'Content-Language': languageHeader,
```
Comment (JLingardAmido) [inline apps/cache-manager/src/app.ts:22]: Is it case sensitive, just weirds me out that the line above has lower case for content-language and here it's upper

**Review by JLingardAmido — COMMENTED** [review]
Can we add a unit test to check it sends back the header if that's what we want it to do and one to check the behaviour of defaulting to 'en-GB' if one is not provided in the request?

File: apps/cache-manager/src/app.ts (hunk 1 above)
Comment (MetecanAydin) [reply apps/cache-manager/src/app.ts:52]: we have a task for it in the project

File: apps/cache-manager/src/app.ts (hunk 2 above)
Comment (MetecanAydin) [reply apps/cache-manager/src/app.ts:22]: don't think it is. made to lower case as well

**Comment by MetecanAydin** [comment]
> Can we add a unit test to check it sends back the header if that's what we want it to do and one to check the behaviour of defaulting to 'en-GB' if one is not provided in the request?

added

**Review by JLingardAmido — APPROVED** [review]

File: apps/cache-manager/src/app.test.tsx [hunk 3]
```diff
@@ -81,7 +98,31 @@ describe('prerender cache manager', () => {
...
+      .set('remote-name', 'testRemote')
+      .set('Content-Language', 'en-GB')
     expect(response.statusCode).toBe(500)
   })
+  it('returns default value for content-language header', async () => {
+    await request(app)
+      .post('/3003/prerender')
+      .send(mockRequestBody)
+      .set('remote-name', 'testRemote')
+      .then(res => {
```
Comment (JLingardAmido) [inline apps/cache-manager/src/app.test.tsx:113]: can we use the await and check the return value like we do for the other tests, good to be consistent and all

Comment (MetecanAydin) [reply apps/cache-manager/src/app.test.tsx]: done

**Review by JLingardAmido — APPROVED** [review]

File: apps/cache-manager/src/app.test.tsx [hunk 4]
```diff
@@ -39,10 +39,15 @@ beforeEach(() => {
 })
 
 describe('prerender cache manager', () => {
-  const base64Body = Buffer.from(JSON.stringify(mockRequestBody)).toString('base64')
+  const base64Body = Buffer.from(JSON.stringify({...mockRequestBody, language: 'en-GB'})).toString('base64')
```
Comment (AlzAmido) [inline apps/cache-manager/src/app.test.tsx:42]: can we have two headers? x-language and x-territory

**Review by AlzAmido — CHANGES_REQUESTED** [review]

**Review by AlzAmido — APPROVED** [review]

**Pull request merged**